torchvision/ops/deform_conv.py

```diff
@@ -50,7 +50,15 @@ def deform_conv2d(
         >>> weight = torch.rand(5, 3, kh, kw)
         >>> # offset and mask should have the same spatial size as the output
         >>> # of the convolution. In this case, for an input of 10, stride of 1
-        >>> # and kernel size of 3, without padding, the output size is 8
+        >>> # and kernel size of 3, without padding, the output size is 8.
+        >>> # For the simplest case, 1 channel 3x3 kernel, at each valid (consider padding) input feature position:
+        >>> # [[p0 p1 p2],
+        >>> #  [p3 p4 p5],
+        >>> #  [p6 p7 p8]]
+        >>> # The 18 corresponding offsets will be:
+        >>> # [p0_offset_h, p0_offset_w, p1_offset_h, p1_offset_w, p2_offset_h, ..., p8_offset_w]
+        >>> # ``_h`` means pixel offset in the height direction (i.e. p0 -> p3 -> p6),
+        >>> # ``_w`` means pixel offset in the width direction (i.e. p0 -> p1 -> p2).
```

Comment:
This part confuses me as well.

I think the indices don't map to the image above, as we have an extra dimension for the `h` and `w`, so users might think that it relates to the `p0` above (which is not actually the case)

Reply:
Maybe something like those images in 
![image](https://user-images.githubusercontent.com/9110200/117962529-9d56c680-b31f-11eb-8b14-8816d0cb02e0.png) the [deformable convolution paper](https://arxiv.org/pdf/1703.06211.pdf), with some code snippets showing something like 
![image](https://user-images.githubusercontent.com/9110200/117962722-d4c57300-b31f-11eb-8a04-2d29c41545ac.png) in the same paper?

Reply:
Yes it have been quite difficult to put this description into words... Maybe an image could be better, with 18 red points like the paper's Figure 6, but it also needs to show which point corresponds to which channel, since that's what #3673 asked.